spec/docs/courses_spec.rb

```diff
@@ -39,9 +39,19 @@
                 required: false,
                 example: { page: 2, per_page: 10 },
                 description: "Pagination options to navigate through the collection."
+      parameter name: :include,
+                in: :query,
+                type: :string,
+                required: false,
+                description: "The associated data for this resource.",
+                schema: {
+                  enum: %w[accredited_body provider recruitment_cycle],
+                },
+                example: "recruitment_cycle,provider"
 
       response "200", "The collection of courses." do
         let(:year) { "2020" }
+        let(:include) { "provider" }
 
```

Comment:
can you create one course for this please. at the moment it passes because there are no courses

Reply:
good spot and done https://github.com/DFE-Digital/teacher-training-api/pull/1534/files#diff-7add9c00508140e920eeff037644b94cR57